llvm/lib/Support/RWMutex.cpp

```diff
@@ -13,7 +13,6 @@
 #include "llvm/Support/RWMutex.h"
 #include "llvm/Config/config.h"
 #include "llvm/Config/llvm-config.h" // for LLVM_ENABLE_THREADS
-#include "llvm/Support/Allocator.h"
```

Comment:
This change seems to break building on macOS

I added it back in 5174d00365b619b5dcd2a451aaf7d844c36ce04d.